perl/dancer2/cpanfile

```diff
@@ -1,4 +1,4 @@
-requires 'Dancer2';
+requires 'Dancer2', '> 0.3';
```

Comment:
Is it correct to right version as is @xsawyerx ?

I mean is there a standard way to say that I want to install any `0.3.x` ?